src/components/books/books.html

```diff
@@ -0,0 +1,12 @@
+<div class="main-container">
+    <div class="search">
+      <input type="text" ng-model="searchText" class="search-area" placeholder="Buscá un libro por titulo o autor ">
+    </div>
+    <div class="detail">
+        <div class="book-container" ng-repeat="book in $ctrl.books | searchBook: searchText">
+          <img class="cover"  ng-src='{{book.image_url ? book.image_url : $ctrl.bookCover}}' alt="Portada" />
+          <p class="book-title"><strong>{{book.title}}</strong></p>
```

Comment:
Instead of the `strong` maybe you can add the `font-weight` to the `book-title` class